security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisResultBuilder.java

```diff
@@ -280,28 +280,21 @@ public SecurityAnalysisResultBuilder endContingency() {
         }
     }
 
-    public class OperatorStrategyResultBuilder extends AbstractLimitViolationsResultBuilder<OperatorStrategyResultBuilder> {
+    public class OperatorStrategyResultBuilder {
 
         private final OperatorStrategy strategy;
 
-        private PostContingencyComputationStatus status = PostContingencyComputationStatus.CONVERGED;
+        private final List<OperatorStrategyResult.ConditionalActionsResult> conditionalActionsResult = Collections.synchronizedList(new ArrayList<>());
+
+        SecurityAnalysisResultContext resultContext;
 
         OperatorStrategyResultBuilder(OperatorStrategy strategy, SecurityAnalysisResultContext resultContext) {
-            super(Objects.requireNonNull(resultContext));
             this.strategy = Objects.requireNonNull(strategy);
+            this.resultContext = resultContext;
         }
 
-        @Override
-        public OperatorStrategyResultBuilder addViolation(LimitViolation violation, SecurityAnalysisResultContext limitViolationContext) {
-            Objects.requireNonNull(limitViolationContext);
-            violations.add(Objects.requireNonNull(violation));
-            //TODO: call to interceptors
-            return this;
-        }
-
-        public OperatorStrategyResultBuilder setStatus(PostContingencyComputationStatus status) {
-            this.status = status;
-            return this;
+        public ConditionalActionsResultBuilder addConditionalActionsResult(String conditionalActionsId) {
```

Comment:
Should be newConditionalActionsResult to be consistent with the others and because at this stage we do not add anything